### PR TITLE
data: Harshitahusts

### PR DESCRIPTION
**Pull request by Harshitahusts**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8729"></a>

### CODE REVIEW

File: data/Harshitahusts.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+  "name": "Harshit Gupta",
+  "displayStatsPublic": ture,
```
Comment (sital002) [inline data/Harshitahusts.json:3]:
```suggestion
  
```
This isnot required anymore. Please remove it.

File: data/Harshitahusts.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  "avatar": "https://github.com/Harshitahusts.png",
```
Comment (sital002) [inline data/Harshitahusts.json:4]:
```suggestion
```

File: data/Harshitahusts.json [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  "bio": "**Open Source** | Java Developer ",
+  "tags": [
+    "Open Source",
+    "JAVA",
+    "MERN",
+    "SPRING",
+    "Docker",
```
Comment (sital002) [inline data/Harshitahusts.json:11]:
```suggestion
    "Docker"
```

File: data/Harshitahusts.json [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+      "icon": "FaGithub",
+      "url": "https://github.com/Harshitahusts"
+    }
+    
+  ],
+  
+  "links": [
+    {
+      "group": "Socials",
+      "name": "Twitter"
```
Comment (sital002) [inline data/Harshitahusts.json:28]:
```suggestion
      "name": "Twitter",
```
Please add comma here

**Review by sital002 — CHANGES_REQUESTED** [review]

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**